Tchap/Modules/Contacts/ContactsViewController.m

```diff
@@ -482,16 +350,6 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         return;
     }
     
-    // Check whether the user wants to add manually some email into the list
-    if ([self.contactsDataSource isAddEmailButtonIndexPath:indexPath])
```

Comment:
@phlniji we have to keep this code to be able to invite by email in an existing room (until we switch to the Element-ios contacts picker)

This button is 
<img width="346" alt="image" src="https://user-images.githubusercontent.com/8969772/164095343-3c5f1f41-d789-465d-ae67-91464cb7a3cf.png">